Automattic/jetpack
# Color update: Update blue and gray illustration colors

#### Changes proposed in this Pull Request

You can see the equivalent update in Calypso here https://github.com/Automattic/wp-calypso/pull/30248

This updates all svg illustration images with the new brand colors from [Color Studio](https://automattic.github.io/color-studio/). This handles only blue and gray colors. I did not touch any Jetpack colors or any color that was not blue or gray.

If the old color was blue, I used the new `Blue` color. If the old color was gray, I used the new `Gray` color.

I basically just searched and replaced hex code colors. I did not open or save the files in any program, so the surrounding html should be unaffected.

Sorry for the large PR. It looks like the diff is not helpful for svg files that have been compressed.

#### Testing instructions

I've checked every image by quickly previewing the local svg file on my machine. Everything looks 👍 



* Check through the images and compared them to master to see if there are any bugs
* I'm not sure where these illustrations are used

Here are some example before/after:

![screen shot 2019-01-24 at 2 46 21 pm](https://user-images.githubusercontent.com/618551/51707521-11ea8780-1fe7-11e9-9726-2f7559c25559.png)
![screen shot 2019-01-24 at 2 45 59 pm](https://user-images.githubusercontent.com/618551/51707524-144ce180-1fe7-11e9-8d62-25d93b439039.png)

Some in context:

![screen shot 2019-01-28 at 1 19 52 pm](https://user-images.githubusercontent.com/618551/51860498-867f3800-22ff-11e9-94c2-1f4f9d56ae95.png)
![screen shot 2019-01-28 at 1 19 47 pm](https://user-images.githubusercontent.com/618551/51860501-88e19200-22ff-11e9-8d9b-b2e3ad79d030.png)

![screen shot 2019-01-28 at 1 21 56 pm](https://user-images.githubusercontent.com/618551/51860583-c2b29880-22ff-11e9-904f-fcb4d83c3c63.png)
![screen shot 2019-01-28 at 1 21 46 pm](https://user-images.githubusercontent.com/618551/51860584-c3e3c580-22ff-11e9-8fea-364626661bc9.png)
